spark-common/src/main/resources/com/k15t/spark/content_iframe_wrapper.vm

```diff
@@ -1,11 +1,10 @@
 ## appending WithHtml to variable name instructs Velocity not to add html escaping (like & -> &amp;)
 ## which would otherwise broke the js embedded this way pretty badly...
 #set($sparkJsWithHtml = $sparkJs)
-<script>
-    ${sparkJsWithHtml}
-</script>
 <div id="${iframeId}-wrapper"></div>
 <script>
+    ${sparkJsWithHtml}
+
     (function(spark) {
         var wrapperEl = document.getElementById("${iframeId}-wrapper");
         spark.appLoader.loadApp(wrapperEl, "${iframeId}", "${iframeSrc}", {
```

Comment:
It should not longer be necessary to pass in that ```openInIframe``` option since it's not evaluated anymore.